src/content/guides/production.md

```diff
@@ -204,7 +205,7 @@ __webpack.prod.js__
 +       'process.env': {
 +         'NODE_ENV': JSON.stringify('production')
 +       }
-+     })
++     }),
```

Comment:
why add a trailing comma? this is inconsistent with the existing code style

Reply:
OK, I’ve looked at `UglifyJSPlugin` here:

![image](https://user-images.githubusercontent.com/2953267/32399716-6641d416-c109-11e7-9db0-5f30f10fbbb0.png)

and thought that if the selected line has a comma in this example, it also has a trailing comma in the example before that – but turns out it doesn’t.

Will update this.